docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -272,23 +292,82 @@
     },
     {
       "metadata": {
-        "id": "To8-oxWUdE2H",
+        "id": "3avZRAzlguOQ",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "// Experiment:\n",
+        "// Use `\\()` to include a floating-point calculation in a string and to include someone's name in a\n",
+        "// greeting."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "4T6rPJ6tWaKu",
         "colab_type": "text"
       },
       "cell_type": "markdown",
       "source": [
-        "#### Experiment\n",
-        "\n",
-        "Use `()` to include a floating-point calculation in a string and to include someone’s name in a greeting.\n",
+        "Use three double quotation marks (`\"\"\"`) for strings that take up multiple lines. Indentation at the start of each quoted line is removed, as long as it matches the indentation of the closing quotation marks. For example:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "_5RJZBV1Webt",
+        "colab_type": "code",
+        "outputId": "8723eb4b-431b-4dcb-bc8c-002c7edc8555",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 54
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "let quotation = \"\"\"\n",
+        "    Even though there's whitespace to the left,\n",
+        "    the actual lines aren't indented.\n",
+        "    Except for this line.\n",
```

Comment:
I'm not sure I understand this line. Looking at the output of the cell, it doesn't look indented.

Reply:
Multiline strings in Swift as aligned based on the indentation of the closing triple quotes.

https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html